packaging/pip/requirements.txt

```
@@ -0,0 +1 @@
+molecule==2.19.0
```

Go for 2.20.1 here, the cookiecutter template needs updated so new work just starts on 2.20
```suggestion
molecule==2.20.1
```